glean-core/android/src/main/java/mozilla/telemetry/glean/testing/GleanTestRule.kt

```diff
@@ -25,6 +26,7 @@ import org.junit.runner.Description
  * val gleanRule = GleanTestRule(ApplicationProvider.getApplicationContext())
  * ```
  */
+@VisibleForTesting(otherwise = VisibleForTesting.NONE)
```

Comment:
Is this required on a-c as well? If yes, would you mind filing a bug/creating the PR? If no, would you mind explaining why?

Reply:
Right now the methods such as `resetGlean` in AC are actually marked as `internal` only, not as `VisibleForTesting`

Reply:
https://github.com/mozilla-mobile/android-components/pull/4024